Makefile

```diff
@@ -117,59 +129,81 @@ test_e2e: generate fmt vet manifests
 manager: generate fmt vet
 	go build -o bin/manager main.go
 
+# it's easier to bump whole kubeconfig instead of grabbing cluster URL from the current context
+_bump_kubeconfig:
+	@mkdir -p $(INTERNAL_TMP_DIR)
+ifndef KUBECONFIG
+	$(eval CONFIG_FILE = ${HOME}/.kube/config)
+else
+	$(eval CONFIG_FILE = ${KUBECONFIG})
+endif
+	cp $(CONFIG_FILE) $(BUMPED_KUBECONFIG)
+
+_login_with_devworkspace_sa:
+	@$(eval SA_TOKEN := $(shell $(K8S_CLI) get secrets -o=json -n $(NAMESPACE) | jq -r '[.items[] | select (.type == "kubernetes.io/service-account-token" and .metadata.annotations."kubernetes.io/service-account.name" == "default")][0].data.token' | base64 --decode ))
+	@echo "Logging as controller's SA in $(NAMESPACE)"
+	oc login --token=$(SA_TOKEN) --kubeconfig=$(BUMPED_KUBECONFIG)
```

Comment:
This now assumes `oc` is installed. Should we do something for `kubectl` as well?

Reply:
Well, fair point but unfortunately, kubectl does not provide such functionality since you how authentication stuff on k8s )
So, instead of relying on kubectl/oc we can generate kubeconfig file by our own, but I think for simplifying we can make oc as prerequisites for make run/debug